proto/vectorindex.proto

```diff
@@ -66,6 +66,81 @@ message _MetadataRequest {
     }
 }
 
+message _AndExpression {
+    _FilterExpression first_expression = 1;
+    _FilterExpression second_expression = 2;
+}
+
+message _OrExpression {
+    _FilterExpression first_expression = 1;
+    _FilterExpression second_expression = 2;
+}
+
+message _NotExpression {
+    _FilterExpression not_expression = 1;
```

Comment:
```suggestion
    _FilterExpression expression_to_negate = 1;
```

Reply:
Okay! Let me also fix this in backend first cause the two proto will have to sync.

Reply:
Done: https://github.com/momentohq/client-protos/pull/246/commits/71fbd3c72bf5ff3d2f3567cf78cbdeebf929bf45